proposals/simd/BinarySIMD.md

```diff
@@ -27,143 +27,143 @@ The `v8x16.shuffle` instruction has 16 bytes after `simdop`.
-| `i32x4.trunc_s/f32x4:sat` |      147 | -                  |
-| `i32x4.trunc_u/f32x4:sat` |      148 | -                  |
-| `i64x2.trunc_s/f64x2:sat` |      149 | -                  |
-| `i64x2.trunc_u/f64x2:sat` |      150 | -                  |
+| `v128.const`              |    `0x00`| i:ImmByte[16]      |
+| `v128.load`               |    `0x01`| m:memarg           |
+| `v128.store`              |    `0x02`| m:memarg           |
+| `i8x16.splat`             |    `0x03`| -                  |
+| `i16x8.splat`             |    `0x04`| -                  |
+| `i32x4.splat`             |    `0x05`| -                  |
+| `i64x2.splat`             |    `0x06`| -                  |
+| `f32x4.splat`             |    `0x07`| -                  |
+| `f64x2.splat`             |    `0x08`| -                  |
+| `i8x16.extract_lane_s`    |    `0x09`| i:LaneIdx16        |
+| `i8x16.extract_lane_u`    |    `0x0a`| i:LaneIdx16        |
+| `i16x8.extract_lane_s`    |    `0x0b`| i:LaneIdx8         |
+| `i16x8.extract_lane_u`    |    `0x0c`| i:LaneIdx8         |
+| `i32x4.extract_lane`      |    `0x0d`| i:LaneIdx4         |
+| `i64x2.extract_lane`      |    `0x0e`| i:LaneIdx2         |
+| `f32x4.extract_lane`      |    `0x0f`| i:LaneIdx4         |
+| `f64x2.extract_lane`      |    `0x10`| i:LaneIdx2         |
+| `i8x16.replace_lane`      |    `0x11`| i:LaneIdx16        |
+| `i16x8.replace_lane`      |    `0x12`| i:LaneIdx8         |
+| `i32x4.replace_lane`      |    `0x13`| i:LaneIdx4         |
+| `i64x2.replace_lane`      |    `0x14`| i:LaneIdx2         |
+| `f32x4.replace_lane`      |    `0x15`| i:LaneIdx4         |
+| `f64x2.replace_lane`      |    `0x16`| i:LaneIdx2         |
+| `v8x16.shuffle`           |    `0x17`| s:LaneIdx32[16]    |
+| `i8x16.add`               |    `0x18`| -                  |
+| `i16x8.add`               |    `0x19`| -                  |
+| `i32x4.add`               |    `0x1a`| -                  |
+| `i64x2.add`               |    `0x1b`| -                  |
+| `i8x16.sub`               |    `0x1c`| -                  |
+| `i16x8.sub`               |    `0x1d`| -                  |
+| `i32x4.sub`               |    `0x1e`| -                  |
+| `i64x2.sub`               |    `0x1f`| -                  |
+| `i8x16.mul`               |    `0x20`| -                  |
+| `i16x8.mul`               |    `0x21`| -                  |
+| `i32x4.mul`               |    `0x22`| -                  |
+| `i8x16.neg`               |    `0x24`| -                  |
+| `i16x8.neg`               |    `0x25`| -                  |
+| `i32x4.neg`               |    `0x26`| -                  |
+| `i64x2.neg`               |    `0x27`| -                  |
+| `i8x16.add_saturate_s`    |    `0x28`| -                  |
+| `i8x16.add_saturate_u`    |    `0x29`| -                  |
+| `i16x8.add_saturate_s`    |    `0x2a`| -                  |
+| `i16x8.add_saturate_u`    |    `0x2b`| -                  |
```

Comment:
Maybe reserve opcodes here as well?

Reply:
Can reserve, did you have something specific in mind to fill the space?

Reply:
I was thinking `{i32x4,i64x2}.add_sat_{s,u}` could logically go here, even if it could not be implemented reasonably today. My thinking is that we should reserve opcodes to be maximally future proof and ensure we have a nice layout no matter how hardware or our priorities change. But if there is no way we would ever want to add these ops, then reserving opcodes for them probably isn't necessary.